[docs] Add a navigation overview (#40321)
expo/expo: docs/pages/develop/app-navigation.mdx

---
title: Navigation in Expo and React Native apps
description: Learn about the recommended approach for integrating navigation in an Expo and React Native project.
sidebar_title: Navigation
---

import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';

import { BoxLink } from '~/ui/components/BoxLink';

The core React Native library does not include a built-in navigation solution, so you can choose a navigation library that best fits your needs. For Expo and React Native apps, it is generally a choice between [React Navigation](https://reactnavigation.org/) or [Expo Router](/router/introduction/).

## Why React Native apps needs a navigation library

> React Native core includes basic UI components, touch handling, device APIs and networking, but excludes, among other things, storage, navigation, camera, maps, most device sensors and navigation, leaving these to community-based libraries.

Reviewer: ```suggestion
React Native core includes basic UI components, touch handling, device APIs and networking, but excludes, among other things, storage, camera, maps, most device sensors - and navigation! These are intended to be covered by community libraries.
```
